tuneme/templates/polls/detail.html

```diff
@@ -9,7 +9,7 @@
 	<div class="icon"><img src="{% static 'img/polls.png' %}" alt="Have Your Say"></div>
 	<h6><a href="">{% trans "Have Your Say" %}</a></h6>
 </div>
-
+{{form.errors}}
```

Comment:
Not needed. You'll need to change the error below on `line 15` to display the error using `form.choice.errors`